pandas/tests/reshape/test_tile.py

```diff
@@ -488,6 +494,34 @@ def test_datetime_cut(self):
         result, bins = cut(data, 3, retbins=True)
         tm.assert_series_equal(Series(result), expected)
 
+    def test_datetimetz_cut(self):
```

Comment:
can you put near the other datetime tests

Reply:
If in another file, any suggestions? There's another datetime cut test right above this one (`test_datetime_cut`)

Reply:
I c. can you move all of the datetime tests to a new Class then